tests/unittest_inference.py

```diff
@@ -3974,6 +3974,71 @@ def test():
         with self.assertRaises(exceptions.AstroidTypeError):
             inferred.getitem(nodes.Const("4"))
 
+    def test_infer_arg_called_type_is_uninferable(self):
+        node = extract_node(
+            """
+        def func(type):
+            type #@
+        """
+        )
+        inferred = next(node.infer())
+        assert inferred is util.Uninferable
+
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="only fails on Python 3.9")
+    def test_infer_arg_called_object_when_used_as_index_is_uninferable(self):
+        node = extract_node(
+            """
+        def func(object):
+            ['list'][
+                object #@
+            ]
+        """
+        )
+        inferred = next(node.infer())
+        assert inferred is util.Uninferable
+
+    @pytest.mark.skipif(sys.version_info < (3, 8), reason="only fails on Python 3.9")
```

Comment:
Is it python 3.9 or python 3.9 and above ? The test or the reason need to be updated :)